example.js

```diff
@@ -0,0 +1,353 @@
+var mutate = require('xtend/mutable')
+var expose = require('choo-expose')
+var css = require('sheetify')
+var html = require('bel')
+var choo = require('./')
+
+css('todomvc-common/base.css')
+css('todomvc-app-css/index.css')
+
+if (module.parent) {
+  exports.todoStore = todoStore
+} else {
+  var app = choo()
+
+  if (process.env.NODE_ENV !== 'production') {
+    var persist = require('choo-persist')
+    var logger = require('choo-log')
+    app.use(persist())
+    app.use(logger())
+  }
+  app.use(expose())
+  app.use(todoStore())
+
+  app.route('/', mainView)
+  app.route('#active', mainView)
+  app.route('#completed', mainView)
+  app.mount('body')
```

Comment:
It's still there, and should be in the README!  - the reason for `app.mount()` is that I like writing my apps as:
```html
<body>
  <header></header>
  <main></main>
  <footer></footer>
</body>
```
`app.mount()` is to replace a DOM node. (e.g. `document.body`), `app.start()` is to append to a DOM node.